includes/shortcodes.php

```diff
@@ -0,0 +1,135 @@
+<?php
+
+function get_person_markup( $person, $title=null ) {
```

Comment:
Functions need to be wrapped in a function check or in a class. If there are multiple functions in a file that are all related, they might as well be wrapped in a class.